Oop/OOP/Fishes.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OOP
+{
+    public class Fishes : Animals
+    {
```

Comment:
ce functionalitate aduce asta in plus?
ca vad ca nu are absolut nimic in plus, e copie fidela de animals..... Si daca e copie fidela..de ce mai ai nevoie de ea?